pkg/gui/keybindings.go

```diff
@@ -361,6 +361,24 @@ func (self *Gui) GetInitialKeybindings() ([]*types.Binding, []*gocui.ViewMouseBi
 			Handler:     self.toggleWhitespaceInDiffView,
 			Description: self.c.Tr.ToggleWhitespaceInDiffView,
 		},
+		{
+			ViewName:    "commits",
```

Comment:
I'm thinking we make this a global keybinding given the number of places it will come in handy. We have a global controller at pkg/gui/controllers/global_controller.go where I'd add the keybinding. In future we can expand it out to be a menu with other diff-based toggles in it. What do you think?

In general, we want newly created keybindings to live in controllers rather than this keybindings file. I want to eventually move everything out of here.

Although as I say this I realise we're calling a function defined in our gui god struct meaning we might need to move some stuff across. Let me know if you need guidance on that!

Reply:
I was also considering making it a global keybinding, but wasn't sure. I don't quite understand what you say about global_controller.go; for now I simply changed the existing keybinding to global, see d838965a4197. This way it ends up right next to "increase/decrease context" in the `x` menu, which is nice.

Reply:
All good, that's fine for now